src/main/java/se/sundsvall/oepintegrator/api/model/cases/ConfirmDeliveryRequest.java

```diff
@@ -0,0 +1,102 @@
+package se.sundsvall.oepintegrator.api.model.cases;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import java.util.Objects;
+
+@Schema(description = "ConfirmDelivery request model")
+public class ConfirmDeliveryRequest {
+
+	@NotBlank
+	@Schema(description = "The case id", example = "123456")
+	private String caseId;
```

Comment:
Isn't `flowInstanceId` the same as "caseId"? 
The flowInstanceId is available as a path param.

Reply:
`flowInstanceId` is the caseId in OpenE.
`caseId` here refers to our caseId. Like `errand.id` in SM or ByggR-caseId. I realize this should have some better naming or at the very least a more detailed description.

Reply:
Yes, maybe a more detailed description then... :-)